learnMoreScreens/TraumaticStressReactions.js

```diff
@@ -49,7 +49,7 @@ class TraumaticStressReactions extends Component {
               )}
         />
         <Text style={styles.subTitle}>
-          <Text style={{ fontWeight: 'bold' }}>{translate('traumaticStressReactions.content.videoCard.title')}</Text>
+          <Text style={{ fontFamily: 'avenir-heavy' }}>{translate('traumaticStressReactions.content.videoCard.title')}</Text>
```

Comment:
Video caption was missing a period. Fixed the language JSON file and pushed fix.